otx/algorithms/action/utils/convert_public_data_to_cvat.py

```diff
@@ -302,18 +326,35 @@ def read_ava_csv(csv_path):
     return annot_info
 
 
-def main(src_path, dst_path, task):
+def parse_args():
```

Comment:
Why does this code take separate inputs?

Reply:
It's WIP, will be handled after @jaegukhyun's update.